nf_core/pipeline-template/{{cookiecutter.name_noslash}}/docs/README.md

```diff
@@ -2,11 +2,11 @@
 
 The {{ cookiecutter.name }} documentation is split into the following files:
 
-1. [Installation](installation.md)
+1. [Installation](https://nf-co.re/usage/installation)
 2. Pipeline configuration
-    * [Local installation](configuration/local.md)
-    * [Adding your own system](configuration/adding_your_own.md)
-    * [Reference genomes](configuration/reference_genomes.md)
+    * [Local installation](https://nf-co.re/usage/local_installation)
```

Comment:
Love this idea of keeping the ToC with external links 👍

Reply:
Thanks - just thought the ToC is really helpful :+1: